### PR TITLE
Update predict-transact-sql.md

### PR DESCRIPTION
**Pull request by arunguru-msft**
ONNX support in Managed Instance is still in preview updating document to reflect this

### CODE REVIEW

**Comment by PRMerger20** [comment]
@arunguru-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/t-sql/queries/predict-transact-sql.md [hunk 1]
```diff
@@ -114,9 +114,9 @@ The DATA parameter is used to specify the data used for scoring or prediction. D
 **RUNTIME = ONNX**
 
 > [!IMPORTANT]
-> The `RUNTIME = ONNX` argument is only available in [Azure SQL Managed Instance](/azure/azure-sql/managed-instance/machine-learning-services-overview), [Azure SQL Edge](/azure/sql-database-edge/onnx-overview), and [Azure Synapse Analytics](/azure/synapse-analytics/overview-what-is).
+> The `RUNTIME = ONNX` argument is only available in [Azure SQL Managed Instance](in Preview)(/azure/azure-sql/managed-instance/machine-learning-services-overview), [Azure SQL Edge](/azure/sql-database-edge/onnx-overview), and [Azure Synapse Analytics](/azure/synapse-analytics/overview-what-is).
 
-Indicates the machine learning engine used for model execution. The `RUNTIME` parameter value is always `ONNX`. The parameter is required for Azure SQL Edge and Azure Synapse Analytics. On Azure SQL Managed Instance, the parameter is optional and only used when using ONNX models.
+Indicates the machine learning engine used for model execution. The `RUNTIME` parameter value is always `ONNX`. The parameter is required for Azure SQL Edge and Azure Synapse Analytics. On Azure SQL Managed Instance(in Preview), the parameter is optional and only used when using ONNX models.
```
Comment (dphansen) [inline docs/t-sql/queries/predict-transact-sql.md:119]:
```suggestion
Indicates the machine learning engine used for model execution. The `RUNTIME` parameter value is always `ONNX`. The parameter is required for Azure SQL Edge and Azure Synapse Analytics. On Azure SQL Managed Instance (in Preview), the parameter is optional and only used when using ONNX models.
```

File: docs/t-sql/queries/predict-transact-sql.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ The `MODEL` parameter is used to specify the model used for scoring or predictio
 ::: moniker range="=azuresqldb-mi-current"
 The `MODEL` parameter is used to specify the model used for scoring or prediction. The model is specified as a variable or a literal or a scalar expression.
 
-In Azure SQL Managed Instance, `PREDICT` supports models in [Open Neural Network Exchange (ONNX)](https://onnx.ai/get-started.html) format or models trained using the [RevoScaleR](../../machine-learning/r/ref-r-revoscaler.md) and [revoscalepy](../../machine-learning/python/ref-py-revoscalepy.md) packages.
+In Azure SQL Managed Instance (In preview), `PREDICT` supports models in [Open Neural Network Exchange (ONNX)](https://onnx.ai/get-started.html) format or models trained using the [RevoScaleR](../../machine-learning/r/ref-r-revoscaler.md) and [revoscalepy](../../machine-learning/python/ref-py-revoscalepy.md) packages.
```
Comment (dphansen) [inline docs/t-sql/queries/predict-transact-sql.md:101]:
```suggestion
In Azure SQL Managed Instance (in preview), `PREDICT` supports models in [Open Neural Network Exchange (ONNX)](https://onnx.ai/get-started.html) format or models trained using the [RevoScaleR](../../machine-learning/r/ref-r-revoscaler.md) and [revoscalepy](../../machine-learning/python/ref-py-revoscalepy.md) packages.
```

File: docs/t-sql/queries/predict-transact-sql.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ helpviewer_keywords:
   - "PREDICT clause"
 author: dphansen
 ms.author: davidph
-monikerRange: ">=sql-server-2017||=azuresqldb-current||>=sql-server-linux-2017||=azuresqldb-mi-current||>=azure-sqldw-latest"
+monikerRange: ">=sql-server-2017||=azuresqldb-current||>=sql-server-linux-2017||>=azure-sqldw-latest"
```
Comment (dphansen) [inline docs/t-sql/queries/predict-transact-sql.md:21]: This moniker still needs to be here to show up in the Azure SQL MI docs.
```suggestion
monikerRange: ">=sql-server-2017||=azuresqldb-current||>=sql-server-linux-2017||=azuresqldb-mi-current||>=azure-sqldw-latest"
```

**Review by dphansen — CHANGES_REQUESTED** [review]
A few minor changes. Please accept these suggestions, and I will go ahead and merge them.

**Comment by PRMerger20** [comment]
@arunguru-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@arunguru-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@dphansen Could you review the updated pull request? Thanks!

**Comment by LitKnd** [comment]
Hello,

Apologies for the delay in handling this. I've emailed @arunguru-msft to follow up on the best course of action to take to resolve this.

Kendra from the Database Docs Team

**Comment by LitKnd** [comment]
Hi,

Per an email thread with @arunguru-msft , I have pulled the changes here into an internal PR, which allows me to also refresh some other areas of the article as well.

Apologies again for the delay on this. I am going to close this PR and get the internal version merged shortly.

Kendra from the Database Docs Team
#please-close

**Pull request closed without merging**